ui/src/components/retreat/home/Schedule.js

```diff
@@ -0,0 +1,96 @@
+import { Box, Container, Flex, Heading, Text } from '@chakra-ui/layout';
+import { RetreatSchedule } from './RetreatSchedule';
+import { MdOutlineEvent } from 'react-icons/md';
+import { Button, Icon } from '@chakra-ui/react';
+import { DateTime } from 'luxon';
+
+const DaySchedule = ({dateString, item}) => {
+  const dateObj = DateTime.fromFormat(dateString, 'dd-MM-yyyy');
+  const scheduleDate = dateObj.toFormat('dd');
+  const scheduleDay = dateObj.toFormat('cccc');
+  
+  const DayDateElement = () => {
+    return (
+      <Flex flexDirection='column' paddingY={{md: 2, lg: 4}} paddingLeft={{md: 2, lg: 4}} width='20%'>
+        <Text fontWeight={800} fontSize={{ base: 12, md: 9, lg: 12}} textAlign='left'>{scheduleDay}</Text>
+        <Text as='b' fontSize={{ base: 30, md: 25, lg: 35}} marginTop='-10px' textAlign='left'>{scheduleDate}</Text>
+      </Flex>
+    );
+  }
+
+  const EventItem = ({title, time, color}) => {
+    return (
+      <Box bg={(color !== '' ? color : '')} borderRadius={8.54} marginBottom={4} border='2px solid #85CCD1'>
+        <Flex flexDir='row' justifyContent='space-between'>
+          <Text as='b' padding={4} fontSize={{base: 15, md: '0.8em', lg: '1em'}}>{title}</Text>
+          <Text as='b' padding={4} fontSize={{base: 15, md: '0.8em', lg: '1em'}}>{time}</Text>
+        </Flex>
+      </Box>
+    )
+  }
+
+  const ScheduleBox = () => {
+    return (
+      <Flex flexDir='column' padding={4} width='80%'>
+        {item.map((event, i) => (
+          <EventItem key={i} title={event.title} time={event.startTime} color={event.color} />
+        ))}
+      </Flex>
+    );
+  }
+
+  return (
+    <Flex flexDirection='column'>
+      <Flex flexDirection='row' justifyContent='space-between'>
+        <DayDateElement />
+        <ScheduleBox />
+      </Flex>
+      <Box bg='#FFDC82' marginX={4} marginBottom={4}><br/></Box>
```

Comment:
should set `ml` or marginLeft to 0 on mobile cos i think this is shifting your box causing it to not be aligned. Also I think the yellow bar should not be present for the final session in the entry
<img width="378" alt="Screenshot 2022-02-22 at 22 49 16" src="https://user-images.githubusercontent.com/16095743/155157114-14b577f8-724e-40b0-b2ff-ac6651fd152f.png">